docs/source/tutorials.rst

```diff
@@ -14,3 +14,4 @@ See tutorials here: https://github.com/MobileTeleSystems/RecTools/tree/main/exam
    examples/5_benchmark_iALS_with_features
    examples/6_benchmark_lightfm_inference
    examples/7_visualization
+   examples/8_baseline_extended_tutorial
```

Comment:
This tutorial is a beautiful piece of work. So we have decided to move it to a proper place.
I'd like to ask you for a few last changes to make it shine more:
1. Let's create a subfolder in `examples` folder in repo with name `tutorials`
2. Let's move your tutorial to this subfolder
3. Let's change the name to `baselines_extended_tutorial.ipynb`
4. In the `tutorials.rst` keep only you tutorial, change the link in the header to tutorials subfolder in the repo
5. Create `examples.rst` where you move all necessary content which is now in `tutorials.rst`

This way we are distinguishing between examples (even long ones) and real theory & practice tutorials.

Reply:
Let's also add `examples` to index.rst